controllers/checlusterbackup/internal_backup_server.go

```diff
@@ -264,16 +301,26 @@ func getInternalBackupServerConfigurationSpec(bctx *BackupContext) *chev1.CheBac
 			Labels:    labels,
 		},
 		Spec: chev1.CheBackupServerConfigurationSpec{
-			Rest: getExpectedInternalRestServerConfiguration(),
+			Rest: getExpectedInternalRestServerConfiguration(bctx),
 		},
 	}
 }
 
-func getExpectedInternalRestServerConfiguration() *chev1.RestServerConfig {
+func getExpectedInternalRestServerConfiguration(bctx *BackupContext) *chev1.RestServerConfig {
```

Comment:
Pls use internal url always

Reply:
Ok, but it means that internal backup server will not be accessible in all namespaces operator mode if internal (across namespaces) communication is not allowed.